static/js/main.js

```diff
@@ -36,6 +36,17 @@
             }
           }
 
+          function validateQuestion(question, value) {
+            var valid = false;
+            if (question.validator == "contact") {
```

Comment:
Unfortunately, yes. The reason is because it's a conditional validator (it needs to validate based on the response of a previous question).